docs/source/getting_started/index.rst

```diff
@@ -4,6 +4,49 @@
 Getting Started
 ===============
 
+Compatibility
+~~~~~~~~~~~~~
+DPF supports Windows 10 and CentOS 7 and later.  For
+more information, see `Ansys Platform Support <https://www.ansys.com/solutions/solutions-by-role/it-professionals/platform-support>`_.
+
+Other platforms may be supported by using DPF within a
+containerization ecosystem such as Docker or Kubernetes.
+For more information, see :ref:`docker`.
```

Comment:
@PProfizi I don't think that the docker page is up to date, we should not add links to it until it is, what do you think?

Reply:
@cbellot000 agreed.